### DISCUSSION TITLE
Collapse component does not close on tab on mobile device

### DISCUSSION THREAD

**Discussion by r-i-s-c** [saadeghi/daisyui#2624] [Q&A]
I am using the collapse checkbox version and everything is working fine on desktop. But on mobile devices the collapse component opens on tab but does not close again on second tab. I can see that the tab gets recognized because the color changes according to the peer-checked property. 
I tried in different browsers on different devices. Using Nextjs. 
So what can cause this problem, is this a bug?

`<div tabIndex={0} className="collapse collapse-plus bg-primary rounded-lg shadow-lg">
    <input type="checkbox" className="peer" />
    <div className="text-xl font-normal text-center collapse-title bg-primary peer-checked:bg-secondary">
      {props.title}
    </div>
    <div className="collapse-content bg-primary peer-checked:bg-secondary">
      <p className='xs:p-1 md:p-4 whitespace-pre-wrap text-md font-light'>{formattedText}</p>
    </div>
    </div>`

**Comment by saadeghi** [comment]
Which mobile device? Which browser version?

**Reply by r-i-s-c** [reply]
iPhone 12 mini, Safari and Chrome

**Reply by saadeghi** [reply]
Which browser version?

**Reply by r-i-s-c** [reply]
Safari 16
